spec/models/agents/freme_filter_agent_spec.rb

```diff
@@ -58,12 +58,23 @@
 
   describe "#receive" do
     before(:each) do
-      @event = Event.new(payload: {data: "Hello from Huginn"})
+      @event = Event.new(payload: {data:
+         <<-END
```

Comment:
I suspected so, but I like specs to be documenting the real API behavior. Since the endpoint expects NIF and I think it would be too difficult to validate the payload being NIF, I wanted at least the spec to document that explicitly :)